lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -705,31 +801,46 @@ def side_effect(*args, **kwargs):
                 'logo_tag': 'Video hosted by XuetangX.com',
                 'url': 'http://www.xuetangx.com'
             },
+            'bumper_metadata': 'null',
             'cdn_eval': False,
             'cdn_exp_group': None,
-            'data_dir': getattr(self, 'data_dir', None),
-            'show_captions': 'true',
-            'handout': None,
             'display_name': u'A Name',
             'download_video_link': None,
-            'end': 3610.0,
+            'handout': None,
             'id': None,
-            'sources': '[]',
-            'speed': 'null',
-            'general_speed': 1.0,
-            'start': 3603.0,
-            'saved_video_position': 0.0,
-            'sub': u'a_sub_file.srt.sjson',
+            'metadata': OrderedDict({
+                "saveStateUrl": "",
+                "autoplay": settings.FEATURES.get('AUTOPLAY_VIDEOS', True),
+                "streams": "1.00:3_yD_cEKoCk",
+                "sub": "a_sub_file.srt.sjson",
+                "sources": '[]',
+                "captionDataDir": None,
+                "showCaptions": "true",
+                "generalSpeed": 1.0,
+                "speed": None,
+                "savedVideoPosition": 0.0,
+                "start": 3603.0,
+                "end": 3610.0,
+                "transcriptLanguage": "en",
+                "transcriptLanguages": OrderedDict({"en": "English"}),
+                "ytTestTimeout": 1500,
+                "ytApiUrl": "www.youtube.com/iframe_api",
+                "ytTestUrl": "gdata.youtube.com/feeds/api/videos/",
```

Comment:
Fixed: "It would be nice to extract a default dict from metadata instead of repeating all the non-changing (key,value)s."